platform/docs/docs/deployment/iframe.md

```diff
@@ -0,0 +1,157 @@
+npx http-server unzipped-folder
+```
+
+You should be able to see
+
+![Alt text](../assets/img/iframe-basic.png)
+
+:::info
+As you see the Cross Origin Isolation Warning is showing up which means we can't render volumes since the volume viewports
+use SharedArrayBuffer which is not allowed non cross origin isolated apps. You can read more about Cross Origin Isolation here
+https://web.dev/coop-coep/ or follow the steps below to enable it.
+:::
+
+### Fixing the Cross Origin Isolation Warning to enable volume rendering
+
+For that we need a more sophisticated setup, since we need to add the Cross Origin Embedder Policy and Cross Origin Opener Policy headers
+to make the parent app cross origin isolated. For that we can use a express server. (Note: you can use any other method
+to add the headers, this is just one of the ways)
+
+Download files from [here](https://ohif-assets.s3.us-east-2.amazonaws.com/iframe-express/Archive.zip)
+
+```js
+const express = require("express")
+const app = express()
+
+app.use((req, res, next) => {
+  res.setHeader("Cross-Origin-Opener-Policy", "same-origin")
+  res.setHeader("Cross-Origin-Embedder-Policy", "require-corp")
+  next()
+})
+
+app.use(express.static("public")) // 'public' should be your folder with static files
+
+app.listen(8080, () => console.log("Listening on port 8080!"))
+```
+
+![Alt text](../assets/img/iframe-headers.png)
+
+
+:::tip
+if you are using webpack with react you can set
+
+```js
+devServer: {
+  headers: {
+    "Cross-Origin-Opener-Policy": "same-origin",
+    "Cross-Origin-Embedder-Policy": "require-corp"
+  }
+}
+```
+
+:::
+
+:::tip
+If you are using Angular, you should modify the `angular.json` file to add the headers
+
+```js
+"serve": {
+  //
+  "configurations": {
+    //
+    "development": {
+      //
+      "headers": {
+        "Cross-Origin-Opener-Policy": "same-origin",
+        "Cross-Origin-Embedder-Policy": "require-corp"
+      }
+    }
+  },
+  //
+},
+```
+:::
+
+
+
+## Development Server
+
+If you are not using the static build, you can use the iframe to load the viewer from the development server. For example, if you are running the viewer locally on port 3000, you can use the following iframe element to load the viewer:
+
+```html
+// e.g., app running on 3001 and iframe loading the viewer from 3000
+<iframe src="http://localhost:3000" style="width: 100%; height: 500px; border: none"/>
+```
+
+You need to notice that not including the static build removes the need for
+the PUBLIC_URL and the routerBasename. However, you need to take care of the Cross Origin Resource Policy (CORP)
+headers since the viewer will be loaded from a different port. You can read more about CORP [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy), but basically in the development server that is serving the viewer, you need to add the following headers:
+
+```js
+"Cross-Origin-Opener-Policy": "same-site"
+or
+"Cross-Origin-Opener-Policy": "cross-origin"
```

Comment:
Add a comment explaining to use this if the embedding app is a completely different origin than OHIF (e.g. parent/embedding app is http://192.168.1.2 and OHIF is http://localhost:3000)

Oh and it should be "Cross-Origin-Resource-Policy" and NOT "Cross-Origin-Opener-Policy"